clang/lib/Driver/Driver.cpp

```diff
@@ -4742,6 +4774,28 @@ class OffloadingActionBuilder final {
     return false;
   }
 
+  void addDeviceDependencies(ActionList &LinkerInputs) {
```

Comment:
The name `addDeviceDependencies` threw me for a loop as it is similar to the existing `addDeviceDepences`.  is there a better name we can use here?

Reply:
Does `addDeviceLinkDependenciesFromHost` look better?